libcloud/dns/drivers/digitalocean.py

```diff
@@ -178,6 +178,10 @@ def create_record(self, name, zone, type, data, extra=None):
                 params['weight'] = extra['weight']
             except KeyError:
                 params['weight'] = 'null'
+            try:
+                params['ttl'] = extra['ttl']
```

Comment:
`params['ttl'] = extra.get('ttl', None)` is a little cleaner

Reply:
:ok_hand: I also thought it would be cleaner to do `extra.get` but I thought the actual try-catch based code was following some coding rules i should follow. Maybe it would be more readable to do this on the others parameters. 

Anyway, I'll edit PR with your suggestion :wink:

Reply:
According to the relevant DigitalOcean API documentations, the `ttl` can not be nulled and I've just tested the change to `extra.get('ttl', None)` and the API response fails with an error 500.

I've looked for a better syntactic way but i think the try-catch structure is the only way to not include this parameter if it does not exists.

Reply:
you could do something like this:
```
if 'ttl' in extra:
    params['ttl'] = extra['ttl']
```

Reply:
Yes ! :man_facepalming:
Thank you !